cirq/contrib/qasm_import/qasm.py

```diff
@@ -17,6 +17,12 @@
 
 
 def circuit_from_qasm(qasm: str) -> circuits.Circuit:
-    """Parses an OpenQASM string to cirq.Circuit."""
+    """Parses an OpenQASM string to cirq.Circuit.
```

Comment:
Nit: Code block for `cirq.Circuit`.
Nit: Add a blank line.

Reply:
Hmmm...I wonder if you meant <code>:obj:&#96;cirq.Circuit&#96;</code> or just <code>&#96;cirq.Circuit&#96;</code>?

Reply:
Either way.  I've never used the first one.  Does it change the documentation?

Reply:
Nothing in the code base uses `:obj:`.

Reply:
`:obj:` adds linking in sphinx docs: https://stackoverflow.com/questions/38968999/python-documentation-objstr-vs-str - but I removed it for consistency.